values/opensuse.yaml

```diff
@@ -35,5 +35,6 @@ packages: >-
     haveged
     tar
     rsync
+    timezone
```

Comment:
why is this needed? Does selinux require this?

Reply:
Good point, not related to SELinux, this is to fix an old glitch we have since the very beginning. Should have noted that on the commit message.

This line here https://github.com/rancher-sandbox/cOS-toolkit/blob/213b48663dc7b96e4cd77446a3e1a77dcf7c97df/packages/cloud-config/oem/01_defaults.yaml#L15

works because `UTC` is always the default, but `systemctl-firstboot` utility expects to find whatever we add here into `/usr/share/zoneinfo/*` and all those timezone definitions are provided by `timezone`. This is like locale setting, defaults to POSIX if you have none installed, but if you choose something else its required to have `glibc-locale` in the system which provides the `/usr/lib/locale/*` files.